src/main/resources/doc/en/html/libs/arith/index.html

```diff
@@ -101,11 +101,75 @@ <h1>
           </td><td>
             <a href="bitfinder.html">Bit Finder</a>
           </td>
+        </tr>
+		 <tr>
+          <td align="right">
+            <a href="adder.html"><img  class="iconlibs" src="../../../../icons/6464/adder.png" alt="#########" width="32" height="32" border="0" ></a>
+          <td>
+            &nbsp;
+          </td><td>
+            <a href="fpadder.html">Floating Point Adder</a>
+          </td>
+        </tr>
+        <tr>
+          <td align="right">
+            <a href="fpsubtractor.html"><img  class="iconlibs" src="../../../../icons/6464/subtractor.png"alt="#########"  width="32" height="32" border="0" ></a>
+          </td><td>
+            &nbsp;
+          </td> <td>
+            <a href="subtractor.html">Floating Point Subtractor</a>
+          </td>
+        </tr>
+        <tr>
+          <td align="right">
+            <a href="fpmultiplier.html"><img  class="iconlibs" src="../../../../icons/6464/multiplier.png" alt="#########" width="32" height="32" border="0"></a>
+          </td><td>
+            &nbsp;
+          </td><td>
+            <a href="fpmultiplier.html">Floating Point Multiplier</a>
+          </td>
+        </tr>
+        <tr>
+          <td align="right">
+            <a href="divider.html"><img  class="iconlibs" src="../../../../icons/6464/divider.png" alt="#########" width="32" height="32" border="0"></a>
+          </td><td>
+            &nbsp;
+          </td><td>
+            <a href="fpdivider.html">Floating Point Divider</a>
+          </td>
+        </tr>
+        <tr>
+          <td align="right">
+            <a href="negator.html"><img  class="iconlibs" src="../../../../icons/6464/negator.png" alt="#########" width="32" height="32" border="0"></a>

```

Comment:
Please remove `alt="#########"` (multiple places). This is absolute no-no. You do not have any content for `alt` then omit it. But do not add content that serves no purpose.

Reply:
For the #####, it is a verification tool that adds them, I thought to put a good information, this in a later stage.
It bothers me too, but it is a detail.